src/openrct2-ui/windows/Finances.cpp

```diff
@@ -511,9 +510,7 @@ namespace OpenRCT2::Ui::Windows
                 || p == WINDOW_FINANCES_PAGE_FINANCIAL_GRAPH)
             {
                 flags |= WF_RESIZABLE;
-                WindowSetResize(
-                    *this, { WW_OTHER_TABS, kHeightOtherTabs },
-                    { std::numeric_limits<int16_t>::max(), std::numeric_limits<int16_t>::max() });
+                WindowSetResize(*this, { WW_OTHER_TABS, kHeightOtherTabs }, { 2000, 2000 });
```

Comment:
Named constants. (Probably just the same as the 5000 ones used by CustomWindow.cpp)